server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1378,6 +1380,9 @@ export class AgenticChatController implements ChatHandlers {
                         }
                         break
                     }
+                    case QCodeReview.toolName:
+                        // no need to write tool message for code review
```

Comment:
Is this a temporary decision? Because to me it would make sense to have a dedicated UI component to show loading states, type of scan in progress, etc rather than just text strings.

Reply:
No, it was decided to not have a dedicated UI component for this currently. It will be just like a conversation. We will continue to use CodeIssues panel for final display of findings.

And we are using the writableStream to show progress of the tool.